app/helpers/template_helpers.py

```diff
@@ -124,31 +211,29 @@ def get_contact_us_url(language_code: str, base_url: str):
     return f"{base_url}contact-us/"
 
 
-def get_footer_urls(language_code: str, base_url: str, schema_theme: str):
+def get_footer_urls(language_code: str, schema_theme: str):
     if language_code == "cy":
-        help_path = "help/sut-i-ateb-y-cwestiynau/help-y-cwestiynau-ar-lein/"
-        cookies_path = "cwcis/"
-        accessibility_statement_path = "hygyrchedd/"
-        privacy_and_data_protection_path = "preifatrwydd-a-diogelu-data/"
-        terms_and_conditions_path = "telerau-ac-amodau/"
+        return {
+            "help_path": "help/sut-i-ateb-y-cwestiynau/help-y-cwestiynau-ar-lein/",
+            "cookies_path": "cwcis/",
+            "accessibility_statement_path": "hygyrchedd/",
+            "privacy_and_data_protection_path": "preifatrwydd-a-diogelu-data/",
+            "terms_and_conditions_path": "telerau-ac-amodau/",
+        }
     else:
-        cookies_path = "cookies/"
-        accessibility_statement_path = "accessibility/"
-        privacy_and_data_protection_path = "privacy-and-data-protection/"
-        terms_and_conditions_path = "terms-and-conditions/"
         help_path = (
             "help/help-with-the-questions/online-questions-help/"
             if schema_theme == "census-nisra"
             else "help/how-to-answer-questions/online-questions-help/"
         )
 
-    return {
-        "help": f"{base_url}{help_path}",
-        "cookies": f"{base_url}{cookies_path}",
-        "accessibility_statement": f"{base_url}{accessibility_statement_path}",
-        "privacy_and_data_protection": f"{base_url}{privacy_and_data_protection_path}",
-        "terms_and_conditions": f"{base_url}{terms_and_conditions_path}",
-    }
+        return {
+            "help_path": help_path,
+            "cookies_path": "cookies/",
+            "accessibility_statement_path": "accessibility/",
+            "privacy_and_data_protection_path": "privacy-and-data-protection/",
+            "terms_and_conditions_path": "terms-and-conditions/",
+        }
```

Comment:
I think the single return was done to ensure this method always returns the a dict with the same properties.